src/core/public/chrome/ui/header/home_icon.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiIcon } from '@elastic/eui';
+import { ChromeBranding } from '../../chrome_service';
+
+/**
+ * Use branding configurations to render the header mark on the nav bar.
+ *
+ * @param {ChromeBranding} - branding object consist of mark, darkmode selection, asset path and title
+ * @returns Mark component which is going to be rendered on the main page header bar.
+ */
+export const HomeIcon = ({
+  darkMode,
+  assetFolderUrl = '',
+  mark,
+  applicationTitle = 'opensearch dashboards',
+  useExpandedMenu = true,
+}: ChromeBranding) => {
+  const { defaultUrl: markUrl, darkModeUrl: darkMarkUrl } = mark ?? {};
+
+  const customMark = darkMode ? darkMarkUrl ?? markUrl : markUrl;
+  const defaultMark = darkMode
+    ? 'opensearch_mark_dark_mode.svg'
```

Comment:
We can export this and check in the tests.

Reply:
done